src/System.Utf8String.Experimental/src/MatchingRefApiCompatBaseline.txt

```diff
@@ -0,0 +1,3 @@
+Compat issues with assembly System.Utf8String.Experimental:
+MembersMustExist : Member 'System.Utf8String.Substring(System.Index)' does not exist in the reference but it does exist in the implementation.
```

Comment:
@GrabYourPitchforks, this should/can be fixed in the ref.

https://github.com/dotnet/coreclr/blob/4ad11ccba4ebb458046891bd07c0c7d4440d7df4/src/System.Private.CoreLib/src/System/Utf8String.Manipulation.cs#L30-L35

Reply:
Unless it was meant to be removed as part of https://github.com/dotnet/coreclr/pull/24036. cc @tarekgh

Reply:
@ahsonkhan yes this method is removed. callers can use Utf8String[Index.GetOffset(Length)..] instead.

Reply:
It is still present in the implementation, thus the baseline. https://github.com/dotnet/coreclr/blob/c2e71fd5e711ca150e238741015a1a0229606793/src/System.Private.CoreLib/src/System/Utf8String.Manipulation.cs#L31